Assets/MRTK/StandardAssets/packagetemplate.json

```diff
@@ -25,5 +25,9 @@
     },
     "bugs": {
         "url": "https://github.com/microsoft/MixedRealityToolkit-Unity/issues"
+    },
+    "dependencies": {
+        "com.thirdparty.mrtk.foundation": "2.5.0",
```

Comment:
This line should be removed as Foundation depends upon StandardAssets.

I will be moving the file(s) at issue from Foundation into StandardAssets in a PR back to your branch.